pkg/client/errors.go

```diff
@@ -0,0 +1,11 @@
+package client
+
+import errorsmod "cosmossdk.io/errors"
+
+var (
+	ErrDial       = errorsmod.Register(codespace, 1, "dialing for connection failed")
+	ErrConnClosed = errorsmod.Register(codespace, 2, "connection closed")
+	ErrSubscribe  = errorsmod.Register(codespace, 3, "failed to subscribe to events")
+	ErrReceive    = errorsmod.Register(codespace, 4, "failed to receive event")
+	codespace     = "query_client"
```

Comment:
This updated to `"events_query_client"` in [ad0121d](https://github.com/pokt-network/poktroll/pull/64/commits/ad0121d63e1a219e551765229bf907731ab28929#diff-a9a023007c887e80b262ad97dfa5092cedfd14e3f02ef3d0662a12dda2925374)